.github/workflows/push_release.yml

```diff
@@ -44,6 +44,15 @@ jobs:
         git config --local user.name "github-actions[bot]"
         poetry run bump2version patch --verbose --commit --no-tag --new-version $VERSION
 
+    - name: Autobump Java version
+      run: |
+        VERSION=$(echo $GITHUB_REF | sed 's#.*/v##')
+        echo $VERSION
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        poetry run bump2version patch --verbose --commit --no-tag --new-version $VERSION
```

Comment:
It's currently at 0.1.5-b0. I think this will make it match the mainline release tag (same as Python package). Alternatively, I could grep the current Java version out of `java/.bumpversion.cfg` and bump that...

Reply:
ok, so going from 0.1.5-b0 -> 0.6.17? We can do that, and sync the versions.

Reply:
Better to have the ability to manage them separately if that is reasonable